libraries/TGRSIAnalysis/TAngularCorrelation/TAngularCorrelation.cxx

```diff
@@ -418,7 +547,8 @@ TGraphAsymmErrors* TAngularCorrelation::CreateGraphFromHst(TH1* hst)
 
       // fill graph with point
       Int_t graphn = graph->GetN();
-      graph->SetPoint(graphn,TMath::Cos(angle),y);
+      if(!fold) graph->SetPoint(graphn,TMath::Cos(angle),y);
+      else if(fold) graph->SetPoint(graphn,angle,y);
```

Comment:
I already noticed this and have fixed it.

----- Original Message -----
From: "SmithJK" <notifications@github.com>
To: "SmithJK/GRSISort" <GRSISort@noreply.github.com>
Cc: "Jennifer Pore" <jpore@sfu.ca>
Sent: Wednesday, March 30, 2016 5:22:39 PM
Subject: Re: [SmithJK/GRSISort] New TAngularCorrelation Updates (#9)

> @@ -418,7 +547,8 @@ TGraphAsymmErrors* TAngularCorrelation::CreateGraphFromHst(TH1* hst)
>  
>        // fill graph with point
>        Int_t graphn = graph->GetN();
> -      graph->SetPoint(graphn,TMath::Cos(angle),y);
> +      if(!fold) graph->SetPoint(graphn,TMath::Cos(angle),y);
> +      else if(fold) graph->SetPoint(graphn,angle,y);

I think you fixed the function output, but not this if statement.

---
You are receiving this because you commented.
Reply to this email directly or view it on GitHub:
https://github.com/SmithJK/GRSISort/pull/9/files/8282b3fbc86e6ca0517763bb035ea67c0e11de14#r57985519